Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -198,6 +198,8 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Inside this method, you should make a call to your backend API to make a charge with that Customer + source, and invoke the `completion` block when that is done.
  *
+ *  @note This method is not called for redirect-based sources. See `paymentContext:redirectDidReturnForSource:`
```

Comment:
Do we still want to add `paymentContext:redirectDidReturnForSource:`? Not sure how necessary it is to expose that.